dashboard/message.go

```diff
@@ -37,8 +37,9 @@ type ChartEntry struct {
 }
 
 type GeneralMessage struct {
-	Version string `json:"version,omitempty"`
-	Commit  string `json:"commit,omitempty"`
+	Version  string `json:"version,omitempty"`
+	Commit   string `json:"commit,omitempty"`
+	SyncMode string `json:"syncMode,omitempty"`
```

Comment:
Pls remove the sync mode, that's mostly an internal detail, is not that useful on the outside, at least not now.